docs/UserGuide.md

```diff
@@ -75,6 +75,9 @@ and efficient Graphical User Interface interaction.
 
 * Parameters can be in any order.<br>
   e.g. if the command specifies `n/NAME p/PHONE`, `p/PHONE n/NAME` is also acceptable.
+  
+* If multiple arguments with same prefixes are in the input and all the values are valid, the last value is chosen. <br>
+  e.g. if the command specifies `ct/w ct/s`, because both `w` and `s` are valid, there is no error, and the type __s - supplier__ is chosen.
```

Comment:
Perhaps we can consider to split up this long sentence as well. Maybe something like:

"if a user's input specifies `ct/w ct/s` where both `w` and `s` are valid, there will be no error thrown. However, the type __s - supplier__ will be chosen instead.